### PR TITLE
Fixed #8

### LINKED ISSUE #8: Rewrite components as functional components
Rewrite as functional components:
* `src/components/App` component
* `src/components/Load ` component

### PR DESCRIPTION
**Pull request by JesseSingleton**
Referencing Issue #8. Rewrote as stateless functional components. ESLint was broken hence the revert. Little bit cleaner.

### CODE REVIEW

**Comment by JesseSingleton** [comment]
Well that was pretty terrible. Might even be worth deleting and making the changes in a separate PR 😢

File: src/components/App.js [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-          <Switch>
-            <Route path="/" exact component={Home} />
-            <Route path="/:user" component={User} />
-          </Switch>
-        </BrowserRouter>
-      </Jumbotron>
-    );
-  }
+const App = () => {
+  return (
```
Comment (ufocoder) [inline src/components/App.js]: You can reduce `return` statement like the followings:
```
const myComp = () => (
  <div>
    ..
  </div>
)
```

**Comment by ufocoder** [comment]
@JesseSingleton Create a new PR or squash commits in the current PR :)

**Comment by nghduc97** [comment]
You can do `git reset --soft` and commit again, that will squash all those commits.

**Pull request closed without merging**